pom.xml

```diff
@@ -62,7 +62,7 @@
         <dependency>
             <groupId>org.gwtproject.regexp</groupId>
             <artifactId>gwt-regexp</artifactId>
-            <version>1.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
reusing this var so that we don't have to change this one place, can just call `mvn versions:set` to change all versions in the project.